tasks/gazette_txt_to_xml.py

```diff
@@ -0,0 +1,239 @@
+    """
+    Creating .zip files for the year's territories 
+    """
+
+    logger.info(f"Gerando ZIP do municipio {xml_file['territory_id']} no ano {xml_file['year']}")
+
+    zip_buffer = BytesIO()
+
+    with ZipFile(zip_buffer, 'w', ZIP_DEFLATED) as zip_file:
+        zip_file.writestr(f"{xml_file['territory_id']}-{xml_file['year']}.xml", xml_file['xml'])
+    
+    zip_size = round(zip_buffer.tell() / (1024 * 1024), 2)
+    zip_buffer.seek(0)
+
+    try:
+        storage.upload_content(zip_path, zip_buffer)
+    except ClientError as e:
+        logger.error(f"Não foi possível fazer o upload do zip do município {xml_file['territory_id']}:\n{traceback.format_exc()}")
+        
+        zip_buffer.close()
+
+    dict_query_info = {
+        "state_code" : xml_file['state_code'],
+        "territory_id" : xml_file['territory_id'],
+        "file_path": zip_path,
+        "year": xml_file['year'],
+        "hash_info": hx,
+        "file_size_mb": zip_size,
+        "last_updated": datetime.utcnow() + br_timezone,
+    }
+
+    database.insert("INSERT INTO aggregates \
+                (territory_id, state_code, year, file_path, \
+                file_size_mb, hash_info, last_updated) \
+                VALUES (%(territory_id)s, %(state_code)s, \
+                %(year)s, %(file_path)s, %(file_size_mb)s, \
+                %(hash_info)s, %(last_updated)s) \
+                ON CONFLICT(file_path) \
+                DO UPDATE \
+                SET state_code=EXCLUDED.state_code, last_updated=EXCLUDED.last_updated, \
+                hash_info=EXCLUDED.hash_info, file_size_mb=EXCLUDED.file_size_mb;", dict_query_info)
+
+    zip_buffer.close()
+
+
+def xml_content_generate(state:str, year:str, territory:dict, list_gazzetes_content:list, storage:StorageInterface):
+    """
+    Generates xml file with gazzetes content  
+    """
+    
+    root = ET.Element("root")
+    xml_file = BytesIO()
+
+    logger.info(f"Gerando XML para cidade {territory['name']}-{state} no ano {year}")
+    
+
+    meta_info_tag = ET.SubElement(root, "meta")
+    ET.SubElement(meta_info_tag, "uf").text = state
+    ET.SubElement(meta_info_tag, "ano_publicacao").text = str(year)
+    ET.SubElement(meta_info_tag, "municipio").text = territory['name']
+    ET.SubElement(meta_info_tag, "municipio_codigo_ibge").text = territory['id']
+    all_gazettes_tag = ET.SubElement(root, "diarios")
+
+    for gazette in list_gazzetes_content:
+        file_gazette_txt = BytesIO()
+        path_arq_bucket = Path(gazette['file_path']).with_suffix(".txt")
+        
+        try:
+            storage.get_file(path_arq_bucket, file_gazette_txt)
+        except ClientError as e:
+            logger.error(f"Erro na obtenção do conteúdo de texto do diário do territorio {gazette['territory_id']}: {e}")
+            file_gazette_txt.close()
+
+            continue
+
+        gazette_tag = ET.SubElement(all_gazettes_tag, "diario")
+        meta_gazette = ET.SubElement(gazette_tag, "meta-diario")
```

Comment:
Para manter a consistência com os outros campos
```suggestion
        meta_gazette = ET.SubElement(gazette_tag, "meta_diario")
```

Reply:
Atualizado!